README.md

```diff
@@ -1,7 +1,8 @@
 # LaunchDarkly Logging API for .NET
 
-[![CircleCI](https://circleci.com/gh/launchdarkly/dotnet-logging/tree/master.svg?style=svg)](https://circleci.com/gh/launchdarkly/dotnet-logging/tree/master)
-[![Documentation](https://img.shields.io/static/v1?label=GitHub+Pages&message=reference&color=00add8)](https://launchdarkly.github.io/dotnet-logging)
+[![NuGet](https://img.shields.io/nuget/v/LaunchDarkly.Logging.svg?style=flat-square)](https://www.nuget.org/packages/LaunchDarkly.Logging/)
+[![CircleCI](https://circleci.com/gh/launchdarkly/dotnet-logging.svg?style=shield)](https://circleci.com/gh/launchdarkly/dotnet-logging)
+[![Documentation](https://img.shields.io/static/v1?label=GitHub+Pages&message=API+reference&color=00add8)](https://launchdarkly.github.io/dotnet-logging)
```

Comment:
This is the same badge format I introduced in https://github.com/launchdarkly/dotnet-server-sdk-redis/pull/15.